homeassistant/components/meteo_france/const.py

```diff
@@ -1,90 +1,127 @@
 """Meteo-France component constants."""
 
 from homeassistant.const import (
+    PRESSURE_HPA,
     SPEED_KILOMETERS_PER_HOUR,
     TEMP_CELSIUS,
-    TIME_MINUTES,
     UNIT_PERCENTAGE,
 )
 
 DOMAIN = "meteo_france"
 PLATFORMS = ["sensor", "weather"]
+COORDINATOR_FORECAST = "coordinator_forecast"
+COORDINATOR_RAIN = "coordinator_rain"
+COORDINATOR_ALERT = "coordinator_alert"
 ATTRIBUTION = "Data provided by Météo-France"
 
 CONF_CITY = "city"
+FORECAST_MODE_HOURLY = "hourly"
+FORECAST_MODE_DAILY = "daily"
+FORECAST_MODE = [FORECAST_MODE_HOURLY, FORECAST_MODE_DAILY]
 
-DEFAULT_WEATHER_CARD = True
+ATTR_NEXT_RAIN_1_HOUR_FORECAST = "1_hour_forecast"
+
+ENTITY_NAME = "name"
+ENTITY_UNIT = "unit"
+ENTITY_ICON = "icon"
+ENTITY_CLASS = "device_class"
+ENTITY_ENABLE = "enable"
+ENTITY_API_DATA_PATH = "data_path"
 
-SENSOR_TYPE_NAME = "name"
-SENSOR_TYPE_UNIT = "unit"
-SENSOR_TYPE_ICON = "icon"
-SENSOR_TYPE_CLASS = "device_class"
 SENSOR_TYPES = {
-    "rain_chance": {
-        SENSOR_TYPE_NAME: "Rain chance",
-        SENSOR_TYPE_UNIT: UNIT_PERCENTAGE,
-        SENSOR_TYPE_ICON: "mdi:weather-rainy",
-        SENSOR_TYPE_CLASS: None,
+    "pressure": {
+        ENTITY_NAME: "Pressure",
+        ENTITY_UNIT: PRESSURE_HPA,
+        ENTITY_ICON: "mdi:gauge",
```

Comment:
Entities with device class set should not set a custom icon. Please set the icon to `None` in these cases.